src/main/java/com/esl/demo/controller/ESLController.java

```diff
@@ -0,0 +1,35 @@
+package com.esl.demo.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/esls")
+public class ESLController implements AbstractController{
+
+    @Override
+    public ResponseEntity getById(Object id) {
```

Comment:
Why return ResponseEntity instead of the actual DTO?

Reply:
As far as I know/ was told this is considered to be "the good practice".

Reply:
Well it depends :). If you can name a few concrete arguments as to why it is "good practice" we can discuss :).